app/models/recipe_ingredient.rb

```diff
@@ -1,3 +1,5 @@
 class RecipeIngredient < ApplicationRecord
-
+belongs_to :recipe 
+belongs_to :ingredient 
+has_many :recipes
```

Comment:
You've got the first 2 associations correct, but we don't want a `has_many :recipes` in a joins table. You just need `belongs_to`